scripts/zones/Temple_of_Uggalepih/mobs/Tonberry_Dismayer.lua

```diff
@@ -15,4 +16,5 @@ function onMobDeath(mob, player, isKiller)
     checkGoVregime(player,mob,793,1);
     checkGoVregime(player,mob,794,1);
     checkGoVregime(player,mob,795,1);
+    phOnDespawn(mob,TONBERRY_KINQ_PH,20,21600); -- 6 hours, 20% pop chance
```

Comment:
20% rate is also pretty unlikely. You don't need to increase the rate to compensate for having a cooldown.